scripts/bash/run_enceladus.sh

```diff
@@ -124,7 +124,11 @@ case $key in
     shift 2 # past argument and value
     ;;
     --delimiter)
-    DELIMITER="\\$2"
+    if [[ "$2" == " " ]]; then
+      DELIMITER="' '"
+    else
+      DELIMITER="\\$2"
```

Comment:
Not very common, but what if there are more than one (special) character in the delimiter, e.g. n spaces? Is it possible to use something like `DELIMITER=\'"${2}"\'`?